lerobot/common/policies/act/modeling_act.py

```diff
@@ -200,15 +208,18 @@ def __init__(self, config: ACTConfig):
         self.config = config
         # BERT style VAE encoder with input tokens [cls, robot_state, *action_sequence].
         # The cls token forms parameters of the latent's distribution (like this [*means, *log_variances]).
-        self.use_input_state = "observation.state" in config.input_shapes
+
+        # If any keys with the prefix "observation.state" are present in the input_shapes, we will use the use state information
+        expected_state_keys = [k for k in config.input_shapes if k.startswith("observation.state")]
```

Comment:
Yeah so as I was mentioning before. Don't you already have this set up as an instance attribute at this point?